### PR TITLE
SCP-4532 Harmonized program names and command-line options.

### PR DESCRIPTION
**Pull request by bwbush**
1. `--...--port-number` becomes `--...--port`.
2. `chainseekd` becomes `marlowe-chain-sync`.
3. References to "chain seek" in environment variables, documentation, and comments become references to "chain sync".
4. No changes to any Haskell type or function names are made.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

**Comment by bwbush** [comment]
@jhbertra , I'm concerned that it's too late to change `chainseekd` to `marlowe-chain-sync` and that this might create chaos. Maybe we should just harmonize the port number flags? However, if we're going to do this, then we shouldn't delay.

**Comment by palas** [comment]
It seems there are still some references to chainseek under the `deploy` folder, I would go all in. But yes @Pacman99, could you confirm that we are not going to break the deployments by changing it? Or if we do where do we need to change it too? 
These other 3 files that also still have references to chainseek:
- `marlowe-integration/src/Test/Integration/Marlowe/Local.hs`
- `marlowe-runtime/tx/Language/Marlowe/Runtime/Transaction/Submit.hs`
- `scripts/setup-dev-connection`

**Comment by bwbush** [comment]
> It seems there are still some references to chainseek . . .

@palas, thanks. I 'll do another grep through the code. In avoiding changing any function or type identifiers, I might have missed so other text.

**Comment by bwbush** [comment]
Hmm, our main branch is changing too fast: each rebase triggers a full re-review of the changes needed.

File: marlowe-chain-sync/docs/marlowe-chain-sync.1 [hunk 1]
```diff
@@ -1,24 +1,24 @@
-.TH chainseekd 1 "18 July, 2022" "version 0.0.0.0"
+.TH marlowe-chain-sync 1 "18 July, 2022" "version 0.0.0.0"
```
Comment (jhbertra) [inline marlowe-chain-sync/docs/marlowe-chain-sync.1:1]: Go ahead and delete all these manpage documents. The have not been maintained and are not used.

**Review by palas — APPROVED** [review]
Thanks, looks good :+1:

File: marlowe-chain-sync/docs/marlowe-chain-sync.1 (hunk 1 above)
Comment (bwbush) [reply marlowe-chain-sync/docs/marlowe-chain-sync.1]: Fixed in 95482d61461f42d5f7abe4850836f26e246b94b3. It looks like we only had one file of this type, but I remember that we used to have more.

**Review by Pacman99 — APPROVED** [review]

**Pull request merged**